lms/djangoapps/student_account/views.py

```diff
@@ -396,6 +442,7 @@ def account_settings_context(request):
         'user_preferences_api_url': reverse('preferences_api', kwargs={'username': user.username}),
         'disable_courseware_js': True,
         'show_program_listing': ProgramsApiConfig.current().show_program_listing,
+        'order_history': get_user_orders(user),
```

Comment:
No the new order history area will not appear right away after this being merged. For that we would need https://github.com/edx/edx-platform/pull/12543 to get merged too.